### DISCUSSION TITLE
How to disable the always-on-top behavior of `dx serve`?

### DISCUSSION THREAD

**Discussion by bluenote10** [DioxusLabs/dioxus#2269] [Q&A]
For desktop apps, `dx serve` seems to implicitly set the always-on-top flag for that window (that is on Linux). To me this behavior is quite annoying, because I cannot "dispatch" the window without actually closing it. Is there a way to prevent making the window always-on-top?

EDIT: Surprisingly in the menu under "Help" there is a "Float on top" setting (sounded to me more like a borderless mode or so). But disabling it is not persisted, i.e., when the window gets opened the next time it is again in always-on-top mode. I'm basically looking for a permanent solution.

**Comment by Lordrin** [comment]
Hi, the answer has been given here: https://github.com/DioxusLabs/dioxus/issues/889#issuecomment-1471900383

I opted to use:
`use_window().window.set_always_on_top(false);`

**Comment by malaire** [comment]
Where do I need to put the `use_window()` call for it to work? I just get error "cannot find function `use_window` in this scope" and Dioxus 0.5 docs are not available at docs.rs currently so can't check them out.

**Comment by DogeDark** [comment]
There's a new CLI setting on the git version of the CLI! You can configure it via `dx config set always-on-top {value}` replacing `{value}` with `true` or `false`

Although unrelated to this discussion, there's a few other settings that can be set which can be viewed with `dx help config set`

**Reply by dabrahams** [reply]
Well, it reports "🚩 CLI setting `always-on-top` has been set. " but it changes nothing in `Dioxus.toml` and doesn't seem to affect the way the app runs. #4544

**Comment by bartoszkp** [comment]
Here is how I'm doing it:

```
fn make_main_window_config() -> Config {
    let main_window = WindowBuilder::new()
        // Other configuration options possible ...
        .with_always_on_top(false);

    dioxus::desktop::Config::default()
        .with_window(main_window)  // Other configuration options possible, e.g. WindowCloseBehavior
}
```

```
fn main() {
    tracing::info!("Launching App.");
    dioxus::LaunchBuilder::desktop()
        .with_cfg(make_main_window_config())
        .launch(App);
}
```
